### PR TITLE
Remove the moved ann_bencmark. No more python dependencies.

### PR DESCRIPTION
**Pull request by aressem**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by toregge — DISMISSED** [review]
👍

**Review by baldersheim — APPROVED** [review]
Yeeeesssss !!! :)

**Review by baldersheim — DISMISSED** [review]
What about pybind ? Seem to remember that was introduced as dependency by ann-benchmark.

File: CMakeLists.txt [hunk 1]
```diff
@@ -52,7 +52,6 @@ find_package(JNI REQUIRED)
 find_package(GTest REQUIRED)
 
 find_package(Python 3.6 COMPONENTS Interpreter Development REQUIRED)
```
Comment (baldersheim) [inline CMakeLists.txt:54]: We require python too, but that should no longer be necessary if we do not need it for anything else.

Comment (aressem) [reply CMakeLists.txt:54]: Testing removal of this dependency in https://github.com/vespa-engine/vespa/pull/28997

**Pull request merged**